XRTK-Core/Packages/com.xrtk.core/Extensions/ArrayExtensions.cs

```diff
@@ -37,12 +37,43 @@ public static bool IsValidArray(this Array array)
         /// <param name="array">The array to extend</param>
         /// <param name="newItem">The item to add to the array</param>
         /// <returns></returns>
-        public static T[] AddItem<T>(this T[] array, T newItem, int insertAtIndex = 0)
```

Comment:
I don't mind keeping this if we're actually inserting it correctly. ATM it doesn't actually insert it at the right index, which is a bug, and why I modified it in the other PR I was working on.

Reply:
We removed the insert at index option for now